compiler-rt/lib/sanitizer_common/sanitizer_stacktrace_printer.h

```diff
@@ -25,7 +26,16 @@ class StackTracePrinter {
  public:
   static StackTracePrinter *GetOrInit();
 
-  virtual const char *StripFunctionName(const char *function) = 0;
+  // Strip interceptor prefixes from function name.
+  const char *StripFunctionName(const char *function);
```

Comment:
Does the `= 0` need to be removed? Same with `RenderSourceLocation` and `RenderModuleLocation`.

Reply:
These are not viritual functions anymore, the implementation of this functions are shared between all the implementation of `StackTracePrinter` abstract class.